flink-dist/src/main/assemblies/opt.xml

```diff
@@ -95,5 +95,12 @@
 			<destName>flink-metrics-statsd-${project.version}.jar</destName>
 			<fileMode>0644</fileMode>
 		</file>
+
+		<file>
+			<source>../flink-metrics/flink-metrics-datadog/target/flink-metrics-datadog-${project.version}.jar</source>
```

Comment:
Can you elaborate more, and give me more guidance on how to do it correctly? 

I copied this piece of code, and saw it exists in flink-metrics-dropwizard/pom.xml, flink-metrics-ganglia/pom.xml, and flink-metrics-graphite/pom.xml.